miner/worker.go

```diff
@@ -1769,6 +1770,12 @@ func (w *worker) commitEx(env *environment, interval func(), update bool, start
 					}
 					logs = append(logs, receipt.Logs...)
 				}
+
+				if update {
+					w.updateSnapshot(env)
```

Comment:
We can avoid calling `w.updateSnapshot()` twice(line 1802) by setting `update` with `false`.

Reply:
i added changing `update` state to false code .👍